src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -309,11 +309,16 @@ class MoreInfoMediaPlayer extends LitElement {
       (state === "playing" &&
         (supportsFeature(stateObj, SUPPORT_PAUSE) ||
           supportsFeature(stateObj, SUPPORT_STOP))) ||
-      (state === "paused" && supportsFeature(stateObj, SUPPORTS_PLAY))
+      (state === "paused" && supportsFeature(stateObj, SUPPORTS_PLAY)) ||
+      (state === "on" &&
+        supportsFeature(stateObj, SUPPORTS_PLAY) &&
+        supportsFeature(stateObj, SUPPORT_PAUSE))
```

Comment:
```suggestion
        supportsFeature(stateObj, SUPPORTS_PLAY) ||
        supportsFeature(stateObj, SUPPORT_PAUSE))
```

Reply:
Line 326 is a bug, we always call `media_play_pause`, even if pause is not supported and we show a stop icon. We should call `media_stop`.

Reply:
Agreed, changed the line as you suggested.